test/physics/em/RelativisticBrem.test.cc

```diff
@@ -0,0 +1,353 @@
+              ElementaryCharge{-1},
+              stable},
+             {"positron",
+              pdg::positron(),
+              MevMass{0.5109989461},
+              ElementaryCharge{1},
+              stable},
+             {"gamma", pdg::gamma(), zero, zero, stable}});
+        const auto& particles = *this->particle_params();
+
+        // Set up shared material data
+        MaterialParams::Input mi;
+        mi.elements  = {{82, AmuMass{207.2}, "Pb"}};
+        mi.materials = {{0.0547 * na_avogadro,
+                         293.15,
+                         MatterState::solid,
+                         {{ElementId{0}, 1.0}},
+                         "Pb"}};
+
+        // Set default material to potassium
+        this->set_material_params(mi);
+        this->set_material("Pb");
+
+        // Set default particle to incident 25 GeV electron
+        this->set_inc_particle(pdg::positron(), MevEnergy{25000});
+        this->set_inc_direction({0, 0, 1});
+
+        // Construct RelativisticBremModel and save the host data reference
+        model_ = std::make_shared<RelativisticBremModel>(
+            ModelId{0}, particles, *this->material_params(), false);
+        data_ = model_->host_ref();
+
+        // Construct RelativisticBremModel and save the host data reference
+        model_lpm_ = std::make_shared<RelativisticBremModel>(
+            ModelId{0}, particles, *this->material_params(), true);
+        data_lpm_ = model_lpm_->host_ref();
+
+        // Set cutoffs: photon energy thresholds and range cut for Pb
+        CutoffParams::Input           input;
+        CutoffParams::MaterialCutoffs material_cutoffs;
+        material_cutoffs.push_back({MevEnergy{0.0945861}, 0.07});
+        input.materials = this->material_params();
+        input.particles = this->particle_params();
+        input.cutoffs.insert({pdg::gamma(), material_cutoffs});
+        this->set_cutoff_params(input);
+    }
+
+    void sanity_check(const Interaction& interaction) const
+    {
+        ASSERT_TRUE(interaction);
+
+        // Check secondaries (bremsstrahlung photon)
+        ASSERT_EQ(1, interaction.secondaries.size());
+
+        const auto& gamma = interaction.secondaries.front();
+        EXPECT_TRUE(gamma);
+        EXPECT_EQ(data_.ids.gamma, gamma.particle_id);
+
+        // Check conservation
+        this->check_conservation(interaction);
+    }
+
+  protected:
+    std::shared_ptr<RelativisticBremModel>       model_;
+    celeritas::detail::RelativisticBremNativeRef data_;
+    std::shared_ptr<RelativisticBremModel>       model_lpm_;
+    celeritas::detail::RelativisticBremNativeRef data_lpm_;
+};
+
+//---------------------------------------------------------------------------//
+// TESTS
+//---------------------------------------------------------------------------//
+TEST_F(RelativisticBremTest, dxsec)
+{
+    const int num_samples         = 10;
+    real_type energy[num_samples] = {1, 2, 5, 10, 20, 50, 100, 200, 500, 1000};
```

Comment:
I'm surprised this doesn't warn, since `const int` isn't the same as `constexpr int`, so you're technically creating a "dynamic" stack object which isn't allowed in C++. You should just be able to use:
```suggestion
    const real_type all_energy[] = {1, 2, 5, 10, 20, 50, 100, 200, 500, 1000};
```
and then change line 150 to `for (real_type energy : all_energy)`, since the value of `i` isn't used except to get the value of the energy array.

Reply:
Fixed as suggested.